### DISCUSSION TITLE
BrowserUtil.Open got an error

### DISCUSSION THREAD

**Discussion by NoobNotFound** [JohnnyCrazy/SpotifyAPI-NET#760] [General]
![image](https://user-images.githubusercontent.com/82730163/165819532-0193dce9-4c78-4765-a56a-82cb4b2bf584.png)
I didn't post this as a issue because this maybe a problem with mine...
My app is using uwp on windows 11.
How to fix this ? Please help 🥺 ...

Thanks

**Comment by FrankBJensen** [comment]
Without your code, its difficult. But have you set up the Spotify callback correctly?

eg:  _server = new EmbedIOAuthServer(new Uri("http://localhost:4002/callback"), 4002);

and you need to set up the same callback (http://localhost:4002/callback) in the Spotify developer enviroment.

Frank.

**Reply by NoobNotFound** [reply]
Thanks for the reply 😄 , I fixed it using by `WebAuthenticationBroker.AuthenticateAsync` 😃
